server/aws-lsp-codewhisperer/src/language-server/telemetry/codeDiffTracker.ts

```diff
@@ -0,0 +1,152 @@
+import { distance } from 'fastest-levenshtein'
+import { Position } from 'vscode-languageserver-textdocument'
```

Comment:
nit: why not import from `'@aws/language-server-runtimes-types'`

Reply:
I can import from server interface instead. (we don't have the runtimes-types in the dependency)